kubernetes/overlays/dev/zookeeper-connect-variable.yaml

```diff
@@ -0,0 +1,15 @@
+
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: kafka-broker
+spec:
+  template:
+    spec:
+      containers:
+        - name: kafka-broker
+          env:
+            - name: KAFKA_ADVERTISED_HOST_NAME
+              value: dev-kafka-cluster
```

Comment:
it would be nice if we could get kustomize to inject the name-prefix for these :thinking: I just couldn't figure it out